content/docs/nav.yml

```diff
@@ -91,63 +91,63 @@
     - id: react-dom-server
       title: ReactDOMServer
     - id: dom-elements
-      title: DOM Elements
+      title: DOM Elemanları
     - id: events
       title: SyntheticEvent
     - id: test-utils
-      title: Test Utilities
+      title: Test Araçları
     - id: shallow-renderer
-      title: Shallow Renderer
+      title: Yüzeysel Render Edici
     - id: test-renderer
-      title: Test Renderer
+      title: Test Render Edici
     - id: javascript-environment-requirements
-      title: JS Environment Requirements
+      title: Javascript Ortam Gereksinimleri
     - id: glossary
-      title: Glossary
-- title: Hooks (New)
+      title: React Terimler Sözlüğü
+- title: Hook (Yeni)
   isOrdered: true
   items:
       - id: hooks-intro
-        title: Introducing Hooks
+        title: Hook'a Giriş
       - id: hooks-overview
-        title: Hooks at a Glance
+        title: Hook'lara Kısa Bir Bakış
       - id: hooks-state
-        title: Using the State Hook
+        title: Hook Durumunu Kullanmak
       - id: hooks-effect
-        title: Using the Effect Hook
+        title: Hook Efekt'ini Kullanmak
       - id: hooks-rules
-        title: Rules of Hooks
+        title: Hook Kuralları
       - id: hooks-custom
-        title: Building Your Own Hooks
+        title: Kendi Hook'unuzu Oluşturun
       - id: hooks-reference
-        title: Hooks API Reference
+        title: Hooks API'si Kaynağı
       - id: hooks-faq
-        title: Hooks FAQ
-- title: Contributing
+        title: Hooks SSS
+- title: Katkı Sağlamak
   items:
   - id: how-to-contribute
-    title: How to Contribute
+    title: Nasıl Katkı Sağlarsanız?
   - id: codebase-overview
-    title: Codebase Overview
+    title: Codebase'e Genel Bakış
   - id: implementation-notes
-    title: Implementation Notes
+    title: Yürütmeye Dair Notlar
```

Comment:
proposal expecting for ` Implementation Notes`

Reply:
I used `uygulama` for `implementation` in many places. We can use `kodlama`, though. Looks fine to me.

>  Uygulamaya Dair Notlar / Kodlama Notları / Hayata Geçirme Notları

Pick your favorite.

Reply:
But it is better to use exactly the same titles with the corresponding page titles. No need to change anything here. Just copy&paste from the docs page. We can review them all later. What do you think ?

Reply:
`Uygulama Notlari` is fine here. Let's just use it